### PR TITLE
VAGOV-1567 - Left Rail Nav from Drupal Menu

### PR DESCRIPTION
**Pull request by acabouet**
## Description
Left rail navigation (for basic pages) built from data pulled from a Drupal menu.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/GetAllPages.graphql.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ module.exports = `
         ... page
       }
     }
+    ${sidebarQuery}
```
Comment (jbalboni) [inline src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: I think we need to add this to `GetLatestPageById`, too?

Comment (beeyayjay) [reply src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: Agreed, @jbalboni. Maybe in GetPageById, too? (I don't know if we're using that one for anything.)

Comment (jbalboni) [reply src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: I don't think we're using that one at the moment, but it wouldn't hurt to add it there, too.

Comment (acabouet) [reply src/site/stages/build/drupal/graphql/GetAllPages.graphql.js]: Just added the sidebar query to GetLatestPageById and GetPageById

**Review by jbalboni — APPROVED** [review]

**Pull request merged**